tests/mock_network/messages.rs

```diff
@@ -77,7 +82,7 @@ fn send_and_receive() {
     // For all expected recipients, poll until it receives the request message and sends
     // the response
     let expected_req_recipients: Vec<_> = expected_recipients(&nodes, &req_dst).collect();
-    assert!(expected_req_recipients.len() >= quorum);
+    assert!(expected_req_recipients.len() >= quorum,);
```

Comment:
this comma is added by autofmt?

Reply:
Yeah, will remove it.

Reply:
just wondering where it comes from, and how to disable it.
I also see the similar stuff couple of times some where else.

Reply:
No idea. Probably rustfmt doesn't handle macros very well.